apps/google-analytics-4/frontend/src/components/main-app/AnalyticsMetricDisplays/AnalyticsMetricDisplay.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ChartFooter from 'components/main-app/ChartFooter/ChartFooter';
+import ChartHeader from 'components/main-app/ChartHeader/ChartHeader';
+import ChartContent from '../ChartContent/ChartContent';
+import { RunReportResponse } from 'types';
+
+interface Props {
+  handleDateRangeChange: Function;
+  runReportResponse: RunReportResponse;
+  reportSlug: string;
+  pageViews: number;
+  metricName: string;
+  error?: Error;
+}
+
+const AnalyticsMetricDisplay = (props: Props) => {
+  const { handleDateRangeChange, runReportResponse, reportSlug, error, metricName, pageViews } =
+    props;
+  return (
+    <>
+      <ChartHeader
+        metricName={metricName ? metricName : ''}
+        metricValue={pageViews || pageViews === 0 ? pageViews.toString() : ''}
```

Comment:
would `pageViews !== undefined` work here?

Reply:
yeah this is a good callout, though I think the way this is written (previously written code I moved into this file) is super explicit about what `pageViews` (a number that can be 0), whereas `pageViews !== undefined` is not as clear to me as to why we are explicitly checking for undefined. Just a personal preference though!

Reply:
I like the way you put that, I completely agree!